bin/generate.py

```diff
@@ -1414,7 +1414,7 @@ def parse_generators(generators_yaml):
         return generators
 
     # Only used at the root directory level.
-    ROOT_KEYS: Final = [
+    ROOT_KEYS: Final[Sequence] = [
```

Comment:
This could be `Final[Sequence[tuple[str, dict[Path, list[Path]], Callable[...something...]]]]`, but I also understand if you don't want to go through the trouble of writing out the entire type :stuck_out_tongue:

Reply:
yeah... i have no idea what the exact type here even is XD